docs/guides/cloud/organizations.mdx

```diff
@@ -338,9 +338,12 @@ organization to sign in through your SSO provider.
 
 #### **Notes**
```

Comment:
We can kill the extra "**" syntax here as well since the `<hX>` tag applies these properties 🙏